addon/Config.jsm

```diff
@@ -37,29 +37,34 @@ var config = {
       * - If there is no key for an endStudy reason, no url will open.
       * - usually surveys, orientations, explanations
       */
+    // TODO change testing survey endings to production survey endings
     "endings": {
       /** standard endings */
       "user-disable": {
-        "baseUrl": "http://www.example.com/?reason=user-disable",
+        "baseUrl": "https://qsurvey.mozilla.com/collab/tp-perception?action=disable&reason=user-disable",
```

Comment:
https://qsurvey.mozilla.com/collab/tp-perception is repeated a lot here. Not sure if it'd make sense to convert into a variable and use template strings or whatever. (a la ``` `${QSURVEY_BASE_URL}?action=disable&reason=user-disable` ```)

Also, in a few other experiments, we passed along the package.json version in the query string. I think the theory is that it made metrics easier or something since you'd be able to tell what XPI version the user was using when they gave feedback. ¯\\\_(ツ)\_/¯

Reply:
Good idea; I'll add the template string.

On the other point, the URL string here gets appended in `ShieldUtils.jsm` with the addon `version` along with a lot of other information. Here's an example when I remove the addon in `about:debugging`:
`https://qsurvey.mozilla.com/collab/tp-perception?action=disable&reason=user-disable&shield=3&study=trackingProtectionMessagingExperiment&variation=pseudo-control&updateChannel=nightly&fxVersion=60.0a1&addon=1.2.0&who=d74dd108-aeca-7d4d-9309-1485933a36fc&testing=1&fullreason=undefined`